packages/contracts/contracts/Interfaces/ICollSurplusPool.sol

```diff
@@ -5,7 +5,14 @@ pragma solidity 0.8.17;
 interface ICollSurplusPool {
     // --- Events ---
 
-    event SurplusCollSharesUpdated(address indexed _account, uint256 _newBalance);
+    event SurplusCollSharesClaimed(address indexed _account, uint256 _collShares);
```

Comment:
does it make sense to add before and after surplus value for `_account`?

Reply:
It's always 100% and they claim all, so the new value will be 0

Reply:
removing this event in favor of identical `CollSharesTransferred`